### ISSUE TITLE
error messages when missing bamSitesToWig requirements

### ISSUE THREAD

**Issue by nsheff** [databio/pepatac#68]
Reported by @cschmidl

if `wigToBigWig` for example is not in PATH, you'd get this error: 

```
Traceback (most recent call last):
  File "./bamSitesToWig.py", line 309, in <module>
    good_chromosomes = ct.run()
  File "/Users/christianschmidl/Library/Python/2.7/lib/python/site-packages/pararead/processor.py", line 476, in run
    results = map(self, nonempties)
  File "./bamSitesToWig.py", line 101, in __call__
    stdin=cutsToWigProcess.stdout)
  File "/usr/local/Cellar/python/2.7.13_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/local/Cellar/python/2.7.13_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

that's hard to understand. we need a better way of testing to make sure stuff is ready to go.

**Comment by jpsmith5** [comment]
Further testing required, but addressed and tested minimally in 45c8f47.  Will check if wigToBigWig is callable, if not, will report to user.

**Comment by nsheff** [comment]
I have a few other changes pending that will also solve this at another point.
